CHANGELOG.md

```diff
@@ -4,6 +4,19 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v5.1.0](https://github.com/voxpupuli/puppet-dhcp/tree/v5.1.0) (2023-12-11)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-dhcp/compare/v5.0.1...v5.1.0)
+
+**Breaking changes:**
+
+- Drop Puppet 6 support [\#273](https://github.com/voxpupuli/puppet-dhcp/pull/273) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
Needs to be v6.0.0 due to the breaking change.

Reply:
Thanks for the feedback. I've updated the PR accordingly.